specification/schemas/CombinedFile.json

```diff
@@ -50,7 +51,6 @@
         "relationships": {
           "type": "object",
           "required": [
-            "shop",
```

Comment:
Why was this removed?

Reply:
Is there a good reason to keep the shop relationship?

Reply:
This schema will not exist anymore once the new RPC endpoint is done, so no need to change anything here.